AUTHORS.md

```diff
@@ -16,15 +16,17 @@
 | gongweibao | Wei-Bao Gong |
 | guru4elephant | Daxiang Dong |
 | Guo Sheng | Sheng Guo |
+| [grygielski](doc/images/paddle_poland_team.jpg)| Adam Grygielski |
 | Haichao-Zhang | Hai-Chao Zhang |
 | hedaoyuan | Dao-Yuan He |
 | helinwang | He-Lin Wang |
 | jacquesqiao | Long-Fei Qiao |
-| jczaja | Jacek Czaja |
+| [jczaja](doc/images/paddle_poland_team.jpg) | Jacek Czaja |
```

Comment:
Why should you put a picture here? Though this picture is very nice.

Reply:
@jczaja Discussed with @raindrops2sea, we could show you picture after the click. But could you move the picture to another address (maybe a url, or your personal repo, or FluidDoc/design/mkldnn?), since it's 2M now, and it's too large for Paddle repo.

Reply:
You can trigger ci use `test=develop test=document_fix` to pass all the CI.

Reply:
Thanks for suggestion. I have moved picture to separate branch (on my fork) to avoid making main Paddle repo bloated.